dbld/images/xenial.dockerfile

```diff
@@ -1,41 +1,40 @@
 FROM ubuntu:16.04
-ARG DISTRO=xenial
-ARG OBS_REPO=xUbuntu_16.04
+LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, "
 
-LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>"
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+ENV LANG C.UTF-8
+ENV OS_PLATFORM xenial
 
-COPY helpers/* /helpers/
```

Comment:
Could you explain the benefits of copying the "helper" files one-by-one?

Reply:
This has been resolved since.